### PR TITLE
IGAPP-1173: Move effect to MapView

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Looks good, tested on real iPhone. Is that the right base branch, though?

**Comment by steffenkleinle** [comment]
> Looks good, tested on real iPhone. Is that the right base branch, though?

Kind of, the poi filter bar needs the svg transformer from @sarahsporck's PR. Won't merge it in her branch though.

**Comment by f1sh1918** [comment]
@steffenkleinle can you please resolve the conflicts. Its hard to review that pr with all these svg transform changes

File: native/src/routes/Pois.tsx [hunk 1]
```diff
@@ -88,7 +82,7 @@ const Pois = ({ pois, language, cityModel, route, navigation }: PoisProps): Reac
     }, RESTORE_TIMEOUT)
   }
   const selectPoiFeature = (feature: PoiFeature | null) => {
-    if (feature && cameraRef.current) {
+    if (feature) {
```
Comment (steffenkleinle) [inline native/src/routes/Pois.tsx:85]: @f1sh1918 just to make sure: This was unnecessary, right?

**Comment by steffenkleinle** [comment]
> @steffenkleinle can you please resolve the conflicts. Its hard to review that pr with all these svg transform changes

@f1sh1918 done

File: native/src/routes/Pois.tsx (hunk 1 above)
Comment (f1sh1918) [reply native/src/routes/Pois.tsx:85]: looks okay to me. couldn't find any issue while testing

**Review by f1sh1918 — APPROVED** [review]
Tested on ios. works fine for me. nice refactoring

**Review by sarahsporck — APPROVED** [review]
Tested on emulated Android and ios simulator.

**Pull request merged**